### ISSUE TITLE
Buying TFTs with BTC (UX issue)

### ISSUE THREAD

**Issue by dmitri-alleo** [threefoldtech/test_feedback#114]
The transaction was successful, but the confirmation screen never appeared, 
After the sending the BTC to the wallet, the screen with the notification that it might take up to 30 minutes stayed on even after the transaction was confirmed and funds were delivered to the wallet

![image](https://user-images.githubusercontent.com/65153287/165478247-ab8662be-094f-4a01-8ecf-3c40d030d96b.png)

![image](https://user-images.githubusercontent.com/65153287/165478258-a7e2520f-32f6-4b6e-91a2-53fc5bc8815c.png)

Otherwise everything works as expected

**Comment by xmonader** [comment]
@sameh-farouk please check and create another issue in the private repo to reference this one

**Comment by sameh-farouk** [comment]
@dmitri-alleo this happened with you only once, or you was able to reproduce the same behavior more than once?
@xmonader can I have access to where this instance runs.
